boards/ZCU104/petalinux_bsp/meta-user/recipes-bsp/u-boot/files/platform-top.h

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Is it sufficient to add `#include <configs/xilinx_zynqmp.h>` to avoid adding our versions of DFU_ALT_INFO_RAM and DFU_ALT_INFO_MMC?

Reply:
That file is the one already in the BSP with the addition of the EEPROM lines so I'm assuming it works better than the generic one?